shared/teams/team/selection-popup.tsx

```diff
@@ -29,6 +29,12 @@ const tabThings = {
   members: 'member',
 }
 
+// How much to bump the list height by when the selection popup is present on mobile
+const heightsMap = {
+  channels: {height: 70},
```

Comment:
It'd be nice to have some dynamic solution here, it's likely that the content won't be static.

Reply:
Hmm, as in measuring the floating thing and then plumbing that here? ick that sounds unfun.

Reply:
The alternative is more unfun I think

Reply:
I'd say we could defer this to another ticket, but my members PR makes this dynamic already :/

Reply:
Okay, I'll do it here when your PR is merged.